test/games/strategy/triplea/ui/TestRoute.java

```diff
@@ -0,0 +1,64 @@
+package games.strategy.triplea.ui;
```

Comment:
I also could change the method to non static... This way I could mock the MapRouteDrawer and call the drawing methods on the mock...